docs/man/docker-images.1.md

```diff
@@ -66,6 +66,11 @@ used in builds use **-a**:
 
     docker images -a
 
+The docker images command used to support the --tree and --dot arguments, which
```

Comment:
Thanks for moving fast. This looks great. There were some tagging issues and tighter structure to be done.

-----> https://gist.github.com/moxiegirl/9659170a92b3b4248c07
Previously, the `docker images` command supported the `--tree` and `--dot` arguments, which
displayed different visualizations of the image data.  Docker core removed this functionality. If you liked this functionality, you can still find it in [the third-party dockviz tool](https://github.com/justone/dockviz).